### PR TITLE
SES CU Main Table Storybook

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket
https://trello.com/b/iebIfrww/ses-dashboard

# Actions
Add new component Header and move Small one inside Cu Table

### CODE REVIEW

File: src/stories/components/custom-table-header/custom-table-header.stories.tsx [hunk 1]
```diff
@@ -1,21 +1,46 @@
...
+
+Header.parameters = {
+  figma: {
+    component:
+      'https://www.figma.com/file/pyaYEjcwF2b5uf9y0vIfIy/SES-Dashboard?node-id=6646%3A61302&t=iDXzm6LhfULmvnWw-4',
+    options: {
+      style: {
+        top: 20,
+        left: 6,
+      },
```
Comment (YasielCabrera) [inline src/stories/components/custom-table-header/custom-table-header.stories.tsx:30]: Please, note that in this case the component and the image don't overlap

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

File: src/stories/components/cu-table-column-initiatives/cu-table-column-initiatives.stories.tsx [hunk 2]
```diff
@@ -1,20 +1,42 @@
...
+
+LightMode.parameters = {
+  figma: {
+    component:
+      'https://www.figma.com/file/pyaYEjcwF2b5uf9y0vIfIy/SES-Dashboard?node-id=2156%3A23054&t=Kn4jCXITHbiZO4YA-4',
+    options: {
+      style: {
+        top: 5,
+        left: -27,
+      },
```
[the remark this answers is not in the document]
Comment (alejandrocabriales) [reply src/stories/components/cu-table-column-initiatives/cu-table-column-initiatives.stories.tsx:26]: you are right

File: src/stories/components/cu-table-column-initiatives/cu-table-column-initiatives.stories.tsx [hunk 3]
```diff
@@ -1,20 +1,42 @@
...
-  initiatives: 0,
+DarkMode.parameters = {
+  figma: {
+    component:
+      'https://www.figma.com/file/pyaYEjcwF2b5uf9y0vIfIy/SES-Dashboard?node-id=2156%3A23054&t=Kn4jCXITHbiZO4YA-4',
+    options: {
+      style: {
+        top: 5,
+        left: -27,
+      },
```
[the remark this answers is not in the document]
Comment (alejandrocabriales) [reply src/stories/components/cu-table-column-initiatives/cu-table-column-initiatives.stories.tsx:39]: you are right

File: src/stories/components/cu-table-column-last-modified/cu-table-column-last-modified.stories.tsx [hunk 4]
```diff
@@ -1,16 +1,98 @@
...
+import type { ComponentMeta } from '@storybook/react';
 
 export default {
   title: 'Components/CUTable/LastModified',
   component: CuTableColumnLastModified,
+
+  parameters: {
+    chromatic: {
+      viewports: [375, 834, 1194, 1440],
+    },
```
[the remark this answers is not in the document]
Comment (alejandrocabriales) [reply src/stories/components/cu-table-column-last-modified/cu-table-column-last-modified.stories.tsx:13]: Relate with this, this component have a small change , in the component when is small, the align text change
https://www.loom.com/share/226560fbfcf64c0bb4dcbfef50e8f7a7
Also I take your recommendation and only show breakpoint with the visual change

File: src/stories/components/cu-table-column-links/cu-table-column-links.stories.tsx [hunk 5]
```diff
@@ -1,41 +1,74 @@
...
+
+Links.parameters = {
+  figma: {
+    component:
+      'https://www.figma.com/file/pyaYEjcwF2b5uf9y0vIfIy/SES-Dashboard?node-id=8853%3A101218&t=8qeht4ZmJlXvVXrb-4',
+    options: {
+      style: {
+        top: -12,
+        left: 960,
+      },
```
[the remark this answers is not in the document]
Comment (alejandrocabriales) [reply src/stories/components/cu-table-column-links/cu-table-column-links.stories.tsx:59]: Done

File: src/stories/components/custom-table-header/custom-table-header.stories.tsx (hunk 1 above)
Comment (alejandrocabriales) [reply src/stories/components/custom-table-header/custom-table-header.stories.tsx:30]: done

File: src/stories/components/cu-table-column-last-modified/cu-table-column-last-modified.stories.tsx [hunk 6]
```diff
@@ -1,16 +1,98 @@
...
+      834: 'https://www.figma.com/file/pyaYEjcwF2b5uf9y0vIfIy/SES-Dashboard?node-id=13146%3A131949&t=hcRG1Rif0SbJoAGa-4',
+      1194: 'https://www.figma.com/file/pyaYEjcwF2b5uf9y0vIfIy/SES-Dashboard?node-id=13140%3A131671&t=hcRG1Rif0SbJoAGa-4',
+      1280: 'https://www.figma.com/file/pyaYEjcwF2b5uf9y0vIfIy/SES-Dashboard?node-id=13140%3A131671&t=hcRG1Rif0SbJoAGa-4',
+    },
+    options: {
+      style: {
+        top: -8,
+        left: 0,
+      },
+    },
```
[the remark this answers is not in the document]
Comment (alejandrocabriales) [reply src/stories/components/cu-table-column-last-modified/cu-table-column-last-modified.stories.tsx]: solve

**Review by YasielCabrera — APPROVED** [review]
LGTM

**Pull request merged**